inc/functions_modcp.php

```diff
@@ -228,7 +252,7 @@ function send_report($report)
 		return $pminfo;
 	}
 
-	return false;
+	return true;
```

Comment:
Any reason why you changed this? First it may break things (plugins) and secondly the doc block is wrong then too.